.github/workflows/build.yml

```diff
@@ -178,10 +182,27 @@ jobs:
       - name: "Install dependencies"
         run: "composer install --no-interaction --no-progress"
 
-      - uses: "actions/download-artifact@v4"
+      - name: "Checkout build-infection"
+        uses: actions/checkout@v5
         with:
-          name: "result-cache-${{ matrix.php-version }}"
-          path: "tmp/"
+          repository: "phpstan/build-infection"
+          path: "build-infection"
+          ref: "1.x"
+
+      - name: "Install build-infection dependencies"
+        working-directory: "build-infection"
+        run: "composer install --no-interaction --no-progress"
+
+      - name: "Configure mutators"
+        run: cp build-infection/resources/infection.json5 .
```

Comment:
ok, here we take the config 1:1 from build-infection

Reply:
Just FYI in phpstan-src I'll also want to add https://github.com/phpstan/phpstan-src/tree/2.1.x/build/PHPStan/Build because there are some tests for that in https://github.com/phpstan/phpstan-src/tree/2.1.x/tests/PHPStan/Build.

So if we're copying the whole config I'd still like to be able to modify it afterward.

Reply:
added a `--source-directory='another/path'` option